File: Daily Pratice Question/C LANGUAGE/Question(30).c
// 30. Write a program to input an integer 'n' and print 
// the sum of all its even digits and the sum of all its odd digits separately.
//  Example : Input: 'n' = 132456, Output: 12, 9
// Explanation:
// The sum of even digits = 2 + 4 + 6 = 12
// The sum of odd digits = 1 + 3 + 5 = 9

// considering the n in array
#include<stdio.h>

int main(){
    
  int arr[]={1,3,2,4,5,6};
    int i;
    int even=0;
    int odd=0;
    int len=sizeof(arr)/sizeof(arr[0]);
    for(i=0;i<len;i++){
        if(arr[i]%2==0){
            even=even+arr[i];
        }else{
            odd=odd+arr[i];
        }
    }
    printf("Addition of even numbers are %d\n",even);
    printf("addition of odd numbers are %d",odd);
    return 0;
}

// considering without array


#include <stdio.h>

int main() {
 int n=132456;
 int rem;
 int even=0;
 int odd=0;
while(n>0){
    rem=n%10;
    if(rem%2==0){
        even=even+rem;
    }else{
        odd=odd+rem;
    }
    n=n/10;
}printf("%d\n",even);
printf("%d\n",odd);
    return 0;
}

File: Daily Pratice Question/JAVASCRIPT/question(30).js
// 30. Write a program to input an integer 'n' and print the sum of all its even 
// digits and the sum of all its odd digits separately. 
// Example : Input: 'n' = 132456, Output: 12, 9

// by taking array
let arr=[1,3,2,4,5,6]
let odd=0
let even=0
for(i=0;i<arr.length;i++){
    if(arr[i]%2==0){
        even+=arr[i]
    }else{
        odd+=arr[i]
    }
}console.log(even)
console.log(odd)


// without taking array
let arr1=132456
let odd1=0
let even1=0
let rem=0
while(arr1>0){
    rem=arr1%10
    if(rem%2==0){
        even1+=rem
    }else{
        odd1+=rem
    }
    arr1=Math.floor(arr1/10)
}
console.log(even1)
console.log(odd1)